config/initializers/opentelemetry.rb

```diff
@@ -1,24 +1,26 @@
 # frozen_string_literal: true
 
-require "opentelemetry/sdk"
-require "opentelemetry/exporter/otlp"
-require "opentelemetry/instrumentation/all"
+if Flipper.enabled?(:opentelemetry)
```

Comment:
Should we opt for an ENV variable or a Feature flag? What could be the optimal approach to handle this, Aboobacker?